routes/index.js

```diff
@@ -2,7 +2,11 @@ const express = require("express");
 const router = express.Router();
 
 const {authRouter} = require("./authRouter");
+const pinRouter = require('./pinRouter');
+
 
 router.use("/auth", authRouter);
+router.use('/pins', pinRouter.router)
```

Comment:
이왕이면 auth에서 사용한 방식과 통일해주세요~
```suggestion
router.use('/pins', pinRouter)
```

Reply:
auth 방식과 통일하여 수정하였습니다.